app/models/game.py

```diff
@@ -16,6 +16,9 @@ def create_game(self, num_players, has_mordred, has_oberon):
         while self.filter(is_started=False, joinable_id=joinable_id):
             joinable_id = ''.join(random.choices(string.ascii_uppercase, k=4))
 
+        if not gen_role_list(num_players, has_mordred, has_oberon):
+            return False
```

Comment:
do you prefer to pass stuff like this up the chain using exceptions, or return values? apparently python has expensive exceptions? but this looks kinda ugly to me

Reply:
either is fine w/ me. Performance doesn't matter here

Reply:
ok, leave it like this then